### PR TITLE
Update article toolbar save button color to match other toolbar icons.

### PR DESCRIPTION
**Pull request by montehurd**

### CODE REVIEW

File: Wikipedia/UI-V5/WMFArticleContainerViewController.m [hunk 1]
```diff
@@ -330,7 +329,8 @@ - (UIBarButtonItem*)tableOfContentsToolbarItem {
 
 - (UIBarButtonItem*)saveToolbarItem {
     if (!_saveToolbarItem) {
-        _saveToolbarItem = [[UIBarButtonItem alloc] initWithImage:[UIImage imageNamed:@"save"] style:UIBarButtonItemStylePlain target:nil action:nil];
+        _saveToolbarItem           = [[UIBarButtonItem alloc] initWithImage:[UIImage imageNamed:@"save"] style:UIBarButtonItemStylePlain target:nil action:nil];
+        _saveToolbarItem.tintColor = [UIColor wmf_logoBlue];
```
Comment (bgerstle) [inline Wikipedia/UI-V5/WMFArticleContainerViewController.m]: this is `wmf_logoBlue` but the other's title uses `wmf_blueTintColor`, is that intentional?

Comment (coreyfloyd) [inline Wikipedia/UI-V5/WMFArticleContainerViewController.m]: We should probably switch the tintColor one to the correct color if it is wrong.

**Comment by montehurd** [comment]
Blue tint consistent across home and article tab bar and article links:
screen shot 2015-10-28 at 2 07 48 pm
screen shot 2015-10-28 at 2 08 25 pm

File: Wikipedia/Categories/UIColor+WMFStyle.m [hunk 2]
```diff
@@ -7,6 +7,7 @@
 //
 
 #import "UIColor+WMFStyle.h"
+//#import "UIColor+WMFHexColor.h"
```
Comment (bgerstle) [inline Wikipedia/Categories/UIColor+WMFStyle.m]: can we remove these comments?

File: www/less/misc.less [hunk 3]
```diff
@@ -5,6 +5,10 @@ a:hover {
     text-decoration: none !important;
 }
 
+a{
+    color: #3366CC;
```
Comment (bgerstle) [inline www/less/misc.less]: can you add a comment saying this is supposed to match `wmf_blueTintColor`?

File: Wikipedia/Categories/UIColor+WMFHexColor.m [hunk 4]
```diff
@@ -13,4 +13,13 @@ + (UIColor*)wmf_colorWithHex:(NSInteger)hex
                            alpha:alpha];
 }
 
+- (NSString*)wmf_hexString {
+    // From: http://stackoverflow.com/a/26341062
+    const CGFloat* components = CGColorGetComponents(self.CGColor);
```
Comment (bgerstle) [inline Wikipedia/Categories/UIColor+WMFHexColor.m]: FYI `UIColor` has `getRed:green:blue:alpha:`

**Pull request merged**